.github/workflows/integration-tests.yml

```diff
@@ -55,6 +55,11 @@ jobs:
           export RUNNER_OS=${{ runner.os }}
           QDRANT_VERSION='v1.7.4' ./tests/integration-tests.sh
         shell: bash
+      - name: Fastembed not installed tests
+        run: |
+          pip3 uninstall fastembed -y
+          pytest tests/test_fastembed.py
```

Comment:
I think we should create a directory for fastembed tests. We may add more files, and with a dedicated directory we won't need to update this script

Reply:
I suggest putting of creating a separate directory until we need it